components/storage/blockdevice/COMPONENT_I2CEE/I2CEEBlockDevice.h

```diff
@@ -169,7 +177,36 @@ class I2CEEBlockDevice : public BlockDevice {
     uint32_t _size;
     uint32_t _block;
 
+    bool _address_is_eight_bit;
+
     int _sync();
+
+    using paged_handler = std::function<int(const bd_addr_t &address, const bd_size_t &length, const uint8_t &deviceAddress)>;
```

Comment:
can this be changed to use `Callback` object instead ? That is our provided std:function replacement.